oeps/archived/oep-0034-bp-lint-amnesty.rst

```diff
@@ -22,6 +22,11 @@ OEP-34: Lint Amnesty
 | `Review Period` | 2019-03-12 - 2021-04-30                                |
 +-----------------+--------------------------------------------------------+
 
+.. attention::
+
+   This decision was a one time decision and is not relevant to the current
+   active project.
```

Comment:
There is a related task in https://github.com/openedx/wg-frontend/issues/131 for doing amnesty on eslint warnings which is actively being worked on, but I'm not sure if it's worth keeping the OEP open for that.